website/docs/r/bfd_multihop_interface_policy.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+subcategory: "Tenant Policies"
+layout: "aci"
+page_title: "ACI: aci_bfd_multihop_interface_policy"
+sidebar_current: "docs-aci-resource-bfd_multihop_interface_policy"
+description: |-
+  Manages ACI BFD Multihop Interface Policy
+---
+
+# aci_bfd_multihop_interface_policy #
+
+Manages ACI BFD Multihop Interface Policy
+
+## API Information ##
+
+* `Class` - bfdMhIfPol
+* `Distinguished Name` - uni/tn-{tn_name}/bfdMhIfPol-{policy_name}
+
+## GUI Information ##
+
+* `Location` - Tenant -> Policies -> Protocol -> BFD Multihop -> Interface Policies
+
+
+## Example Usage ##
+
+```hcl
+resource "aci_bfd_multihop_interface_policy" "example" {
+  tenant_dn             = aci_tenant.example.id
+  name                  = "example"
+  admin_state           = "enabled"
+  annotation            = "orchestrator: terraform"
```

Comment:
Do we need annotation in the example? Also if we do should we use either a completely different annotation from the default annotation when not provided, because this one almost matches it.

Reply:
i will remove annotation since it's not a required argument.